jobs/integr8ly/nightly-pipeline-qe-pony.yaml

```diff
@@ -0,0 +1,193 @@
+                            // this patches the webapp to use master image (v2.2.0 is used otherwise)
+                            sh """
+                                oc login https://master.${YOURCITY}.openshiftworkshop.com -u admin@example.com -p Password1  --insecure-skip-tls-verify
+                                oc patch deploymentconfig/tutorial-web-app -n webapp -p '{ "spec": { "template": { "spec": { "containers": [{ "name": "tutorial-web-app", "image": "quay.io/integreatly/tutorial-web-app:master" }]}}}}' || true
+                            """
+                        }
+                    } // stage
+
+                    stage('Smoke Tests') {
+                        buildStatus = build(job: 'installation-smoke-tests', propagate: false, parameters: [
+                            string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                            string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                            string(name: 'ADMIN_USERNAME', value: 'admin@example.com'),
+                            string(name: 'ADMIN_PASSWORD', value: 'Password1')]).result
+                                
+                        println "Build finished with ${buildStatus}"
+                            
+                        if(buildStatus != 'SUCCESS') {
+                            currentBuild.result = 'UNSTABLE'
+                        }
+                    } // stage
+
+                    stage('Browser Based Tests') {
+                        CLUSTER_URI = "${CLUSTER_URL}".replace("https://","")
+                        buildStatus = build(job: 'browser-based-testsuite-pipeline', propagate: false, parameters: [
+                            string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                            string(name: 'WEBAPP_URL', value: "https://tutorial-web-app-webapp.${CLUSTER_URI}"),
+                            string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+                            string(name: 'ADMIN_USERNAME', value: 'admin@example.com'),
+                            string(name: 'ADMIN_PASSWORD', value: 'Password1'),
+                            string(name: 'EVALS_USERNAME', value: 'evals11@example.com')]).result
+                            
+                        println "Build finished with ${buildStatus}"
+                            
+                        if(buildStatus != 'SUCCESS') {
+                            currentBuild.result = 'UNSTABLE'
+                        }
+                            
+                        sleep time: 3, unit: 'MINUTES'
+                    } // stage
+
+                    stage('After First Login Tests') {
+                        buildStatus = build(job: 'after-first-login-tests', propagate: false, parameters: [
```

Comment:
since recently there is a new parameter used for this pipeline: **EVALS_USERNAME**

Reply:
yeah this needs to be updated.